esp-box/include/bsp/touch.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+/**
+ * @file
+ * @brief BSP Touchscreen
+ *
+ * This file offers API for basic touchscreen initialization.
+ * It is useful for users who want to use the touchscreen without the default Graphical Library LVGL.
+ *
+ * For standard LCD initialization, you can call all-in-one function bsp_display_start().
```

Comment:
```suggestion
 * For standard LCD initialization with LVGL graphical library, you can call all-in-one function bsp_display_start().
```
Could you change it in `display.h` too, please?

Reply:
Same for all same files...